db/tests/tables/operations/test_split.py

```diff
@@ -22,18 +24,17 @@ def test_extract_columns_from_table_creates_tables(engine_with_roster, roster_ta
     t_dict = metadata.tables
     assert (
         f"{schema}.{teachers_table_name}" in t_dict
-        and f"{schema}.{roster_no_teachers_table_name}" in t_dict
     )
 
 
 def test_extract_columns_from_table_sets_up_one_fkey(extracted_remainder_roster):
-    extracted, remainder, _, _, _ = extracted_remainder_roster
+    extracted, remainder, _, _ = extracted_remainder_roster
```

Comment:
`extracted` isn't accessed. While I'm not too fussy about that in tests, I think it's a bit confusing given that you're using `_` for other unaccessed return values.